### DISCUSSION TITLE
部分符号规范化结果问题

### DISCUSSION THREAD

**Discussion by caoyicheng11** [SWHL/ProcessLaTeXFormulaTools#4] [Q&A]
用NormalizeFormula会把\sin变成\operatorname { s i n }，\lim变成\operatorname* { l i m }。但我希望获得的公式数据是：\sin和\lim。

**Comment by SWHL** [comment]
你可能需要更改这个地方：https://github.com/SWHL/ProcessLaTeXFormulaTools/blob/ef62ef8643a61f8969eef130b0fae28d9c36e76f/process_formula/preprocess_latex.js#L261-L282

不过建议在NormalizeFormula之后，做一些简单字符串替换更好一些
